src/lib/logger.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-module.exports = require('./httpd').log;
+const logger = require('../lib/httpd').log;
+const bunyan = require('bunyan');
+
+// disable default bunyan stream to stdout
+logger.level(bunyan.FATAL + 1);
+
+function modifiedStream() {
```

Comment:
Instead of this being a separate function, you could pass it directly to `addStream()`. Something like:

```
stream: () => {
     // your function body
    write: entry => {
    ...
}()

Reply:
can't seem to get this one in a working state, I keep hitting issues on:

```
node:assert:400
    throw err;
    ^

AssertionError [ERR_ASSERTION]: "stream" stream is not writable: [Function: stream]
    at Logger.addStream (/Users/meharc/projects/hmpo/lev/lev-api/node_modules/bunyan/lib/bunyan.js:580:16)
    at Object.<anonymous> (/Users/meharc/projects/hmpo/lev/lev-api/src/lib/httpd.js:41:11)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/Users/meharc/projects/hmpo/lev/lev-api/src/app.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1198:14) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
```

Reply:
@aarongill1 thankfully helped resolve this